src/Umbraco.Web/Install/InstallSteps/NewInstallStep.cs

```diff
@@ -138,7 +137,7 @@ public override bool RequiresExecution(UserModel model)
 
             // In this one case when it's a brand new install and nothing has been configured, make sure the
             // back office cookie is cleared so there's no old cookies lying around causing problems
-            _http.ExpireCookie(_umbracoSettingsSection.Security.AuthCookieName);
+            _httpContextAccessor.HttpContext.ExpireCookie(_umbracoSettingsSection.Security.AuthCookieName);
```

Comment:
Was this meant to use the new ICookieManager.ExpireCookie?

Reply:
Yep, good catch